### ISSUE TITLE
training beyond specified 't_total'. Learning rate multiplier set to 0.0. Please set 't_total' of WarmupLinearSchedule cor

### ISSUE THREAD

**Issue by dh12306** [wenyu332/daguan_bert_ner#12]
您好，下了您的代码，还有训练的数据，改了 pretraining_args 里的num_train_epochs = 3，想训练一下模型，然后输出预测，结果，每个epoch的时候是正常的，但是到了epoch=3时，训练还是没停，报错如下：
training beyond specified 't_total'. Learning rate multiplier set to 0.0. Please set 't_total' of WarmupLinearSchedule correctly

![image](https://user-images.githubusercontent.com/20471681/63424269-a4012400-c440-11e9-8d0e-761959dc8a75.png)

然后打印了t_total，就是 num_train_optimization_steps：
![image](https://user-images.githubusercontent.com/20471681/63424378-df035780-c440-11e9-8f17-307d57a0fe74.png)

**Comment by wenyu332** [comment]
这个是因为优化器的warn_up是根据   样本总数/batch_size 来累加的  你可以看一下optimization.py中的_LRSchedule类，里面get_lr方法有显示保错的原因
